src/test/java/org/folio/dew/batch/bursarfeesfines/service/BursarFilterEvaluatorTest.java

```diff
@@ -0,0 +1,359 @@
+      .item(null)
+      .build();
+
+    assertThat(
+      BursarFilterEvaluator.evaluate(
+        accountWithAncillaryData,
+        filterFeeFineOwner
+      ),
+      is(true)
+    );
+  }
+
+  @Test
+  void testFilterAccountByItemLocation() {
+    UUID itemLocationID = UUID.fromString(
+      "00000000-0000-4000-8000-000000000000"
+    );
+    BursarExportFilterLocation filterLocation = new BursarExportFilterLocation();
+    filterLocation.setLocationId(itemLocationID);
+
+    Item item = new Item();
+    ItemLocation itemLocation = new ItemLocation();
+    itemLocation.setId("00000000-0000-4000-8000-000000000000");
+    item.setEffectiveLocation(itemLocation);
+
+    AccountWithAncillaryData accountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .account(null)
+      .user(null)
+      .item(item)
+      .build();
+
+    assertThat(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterLocation),
+      is(true)
+    );
+  }
+
+  @Test
+  void testFilterAccountByPatronGroup() {
+    UUID patronGroupID = UUID.fromString(
+      "00000000-0000-4000-8000-000000000000"
+    );
+    BursarExportFilterPatronGroup filterPatronGroup = new BursarExportFilterPatronGroup();
+    filterPatronGroup.setPatronGroupId(patronGroupID);
+
+    User user = new User();
+    user.setPatronGroup("00000000-0000-4000-8000-000000000000");
+
+    AccountWithAncillaryData accountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .account(null)
+      .user(user)
+      .item(null)
+      .build();
+
+    assertThat(
+      BursarFilterEvaluator.evaluate(
+        accountWithAncillaryData,
+        filterPatronGroup
+      ),
+      is(true)
+    );
+  }
+
+  @Test
+  void testFilterAccountByServicePoint() {
+    UUID servicePointID = UUID.fromString(
+      "00000000-0000-4000-8000-000000000000"
+    );
+    BursarExportFilterServicePoint filterServicePoint = new BursarExportFilterServicePoint();
+    filterServicePoint.setServicePointId(servicePointID);
+
+    Item item = new Item();
+    item.setInTransitDestinationServicePointId(
+      "00000000-0000-4000-8000-000000000000"
+    );
+
+    AccountWithAncillaryData accountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .account(null)
+      .user(null)
+      .item(item)
+      .build();
+
+    assertThat(
+      BursarFilterEvaluator.evaluate(
+        accountWithAncillaryData,
+        filterServicePoint
+      ),
+      is(true)
+    );
+  }
+
+  @Test
+  void testNegationFilter() {
+    BursarExportFilterNegation filterNegation = new BursarExportFilterNegation();
+    filterNegation.setCriteria(new BursarExportFilterPass());
+
+    AccountWithAncillaryData accountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .account(null)
+      .user(null)
+      .item(null)
+      .build();
+
+    assertThat(
+      BursarFilterEvaluator.evaluate(accountWithAncillaryData, filterNegation),
+      is(false)
```

Comment:
Same idea here; we need to test a case where it returns false (feel free to use any inner filter, whatever is easiest)